### ISSUE TITLE
GCPy installation from PyPI (via Pip) does not work due to not finding xesmf version 0.5.1

### ISSUE THREAD

**Issue by yantosca** [geoschem/gcpy#326]
### Your name

Bob Yantosca

### Your affiliation

Harvard + GCST

### What happened? What did you expect to happen?

I get this error when trying to install the `geoschem-gcpy` package from PyPI with pip:
```python
[ip-172-31-41-231 ~]$ pip install geoschem-gcpy
Collecting geoschem-gcpy
  Downloading geoschem_gcpy-1.5.0-py3-none-any.whl (1.3 MB)
     |████████████████████████████████| 1.3 MB 25.5 MB/s
Collecting joblib==1.3.2
  Downloading joblib-1.3.2-py3-none-any.whl (302 kB)
     |████████████████████████████████| 302 kB 108.2 MB/s
Requirement already satisfied: docutils==0.16 in /usr/lib/python3/dist-packages (from geoschem-gcpy) (0.16)
ERROR: Could not find a version that satisfies the requirement xesmf==0.5.1 (from geoschem-gcpy) 
(from versions: 0.1.1, 0.1.2, 0.2.0, 0.2.1, 0.2.2, 0.3.0, 0.7.1, 0.8, 0.8.1, 0.8.2, 0.8.3, 0.8.4, 0.8.5)
ERROR: No matching distribution found for xesmf==0.5.1 (from geoschem-gcpy)
```
The installation fails because `xesmf==0.5.1` (which is used by GCPy) was removed from the PyPI index.

### What are the steps to reproduce the bug?

```console
$ pip install geoschem-gcpy
```

### What GCPy version were you using?

1.5.0

### What environment were you running GCPy on?

AWS

### What python environment were you using?
```console
Package                Version
 ---------------------- -------------------- 
asn1crypto             0.24.0               
attrs                  19.3.0               
Automat                0.8.0               
 awscli                 1.18.69              
blinker                1.4                  
botocore               1.16.19              
certifi                2019.11.28           
chardet                3.0.4                
Click                  7.0                 
 cloud-init             23.2.2               
colorama               0.4.3                
command-not-found      0.3                  
configobj              5.0.6                
constantly             15.1.0               
cryptography           2.8                  
dbus-python            1.2.16               
distro                 1.4.0                
distro-info            0.23+ubuntu1.1       
docutils               0.16                 
entrypoints            0.3                 
 hibagent               1.0.1                
httplib2               0.14.0               
hyperlink              19.0.0               
idna                   2.8                  
importlib-metadata     1.5.0                
incremental            16.10.1              
Jinja2                 2.10.1               
jmespath               0.9.4               
 jsonpatch              1.22                 
jsonpointer            2.0                  
jsonschema             3.2.0                
keyring                18.0.1               
language-selector      0.1                 
 launchpadlib           1.10.13              
lazr.restfulclient     0.14.2               
lazr.uri               1.0.3                
MarkupSafe             1.1.0                
more-itertools         4.2.0                
netifaces              0.10.4               
oauthlib               3.1.0                
olefile                0.46                 
PAM                    0.4.2                
pexpect                4.6.0                
Pillow                 7.0.0                
pip                    20.0.2               
pyasn1                 0.4.2                
pyasn1-modules         0.2.1                
Pygments               2.3.1                
PyGObject              3.36.0               
PyHamcrest             1.9.0               
 PyJWT                  1.7.1                
pyOpenSSL              19.0.0              
 pyrsistent             0.15.5               
pyserial               3.4                  
python-apt             2.0.1+ubuntu0.20.4.1 
python-dateutil       2.7.3                
python-debian          0.1.36+ubuntu1.1     
PyYAML                 5.3.1                
requests               2.22.0               
requests-unixsocket    0.2.0               
roman                  2.0.0                
rsa                    4.0                  
s3transfer             0.3.3                
SecretStorage          2.3.1                
service-identity       18.1.0               
setuptools             45.2.0               
simplejson             3.16.0               
six                    1.14.0               
sos                    4.5.6                
ssh-import-id          5.10                 
systemd-python         234                  
Twisted                18.9.0               
ubuntu-advantage-tools 8001                 
ufw                    0.36                 
unattended-upgrades    0.1                  
urllib3                1.25.8               
wadllib                1.3.3                
wheel                  0.34.2               
zipp                   1.0.0                
zope.interface         4.7.1               
```
### Will you be addressing this bug yourself?

Yes

### Additional information

A fix for this will require updating GCPy to use one of the "approved" versions of xesmf.  Until then, users should follow these instructions:

- https://gcpy.readthedocs.io/en/stable/Getting-Started-with-GCPy.html#for-developers-install-gcpy-from-git

**Comment by yantosca** [comment]
Fixed by PR #329

**Comment by yantosca** [comment]
We can close out this issue as PR #329 (which updates package versions) has now been merged into the GCPy 1.6.0 development stream.
